templates/layouts/_questionnaire.html

```diff
@@ -37,4 +37,12 @@
           })
       }}
   {% endblock %}
+
+  {% block cant_complete %}
+     {% if display_return_to_hub_link %}
+      <p class="u-mt-xl">{{ _("Can't complete this question?") }}<br>
+      <a href="/questionnaire">{{ _("Choose another section and return to this later") }}</a></p>
```

Comment:
This probably shouldn't be hardcoded and prob should use `url_for` or get passed the URL?